SR2019/2019-03-27-final-competition-details.md

```diff
@@ -0,0 +1,25 @@
+---
+to: Student Robotics 2019
+subject: Final competition details
+---
+
+Hi all,
+
+The competition is now less than 2 weeks away! We have some additional important information for you.
+
+Most importantly is media consent. Media consent forms must be sent to us by 5pm Thursday 4th April. After then most of our volunteers will be on site and not handling
+forms. If you've not returned a media consent form by then, it will take significantly longer to pass through reception, and some competitors may be turned away.
+
+You can download the media consent form from [our website](https://studentrobotics.org/events/sr2019/competition/).
+
+Parents and other visitors do not require tickets, but are required to sign media consent forms. These forms should be bought with them, or can be signed at reception.
```

Comment:
s/bought/brought/
(You've made this typo before, is your keyboard's `r` key ok?)

Reply:
https://github.com/srobo/team-emails/pull/19/commits/194acd352a6742f9c5e64c553398619cd29a5d98